docs/book/features/cns_telemetry.md

```diff
@@ -0,0 +1,42 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD034 -->
+
+# vSphere CSI Driver - Cluster Distribution
+
+- [Introduction](#introduction)
+- [Deploy vSphere CSI driver with cluster-distribution](#vsphere-conf-cluster-distribution)
+- [Limitations](#limitations)
+
+**Note:** Feature to support `cluster-distribution` parameter in vSphere CSI driver is released with [v2.2.0](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.2.0).
```

Comment:
since v2.2.0 is not released, let's not provide any link and change the wording as we have published in https://vsphere-csi-driver.sigs.k8s.io/features/vsphere_csi_vmc.html

Reply:
Removed. Will add it back after release.